consensus/src/chained_bft/liveness/pacemaker_test.rs

```diff
@@ -28,12 +32,13 @@ fn test_pacemaker_time_interval() {
 #[test]
 /// Verify that Pacemaker properly outputs local timeout events upon timeout
 fn test_basic_timeout() {
+    let mut runtime = consensus_runtime();
```

Comment:
this probably can be simple tokio::test

Reply:
`consensus_runtime` does a few other things like setting up a logger if --nocapture is passed so while we probably could move to using `tokio::test` we'd loose that behavior. Another issue would be if any code expects to have a handle directly onto a runtime (which you can't get with the test macro) Course there's nothing stopping us from building our own macro to just do that stuff too.

Reply:
These tests didn't have a runtime before my change, so in the end the behavior is unchanged. I agree though, and as soon as we can get consensus more cleaned up, we can put a #[consensus::test] macro to use that does all these nice things.